src/rard/research/migrations/0063_work_book_introductions.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 3.2 on 2023-06-02 09:11
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("research", "0062_definite_possible_clarity"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="book",
+            name="introduction",
+            field=models.OneToOneField(
+                "TextObjectField",
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="introduction_for_book",
+            ),
+        ),
+        migrations.AddField(
+            model_name="work",
+            name="introduction",
+            field=models.OneToOneField(
+                "TextObjectField",
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="introduction_for_work",
+            ),
+        ),
+        migrations.AddField(
+            model_name="book",
+            name="plain_introduction",
+            field=models.TextField(default=""),
+        ),
+        migrations.AddField(
+            model_name="historicalbook",
+            name="plain_introduction",
+            field=models.TextField(default=""),
+        ),
+        migrations.AddField(
+            model_name="historicalwork",
+            name="plain_introduction",
+            field=models.TextField(default=""),
+        ),
+        migrations.AddField(
+            model_name="work",
+            name="plain_introduction",
+            field=models.TextField(default=""),
+        ),
+        migrations.AddField(
+            model_name="historicalbook",
+            name="introduction",
+            field=models.ForeignKey(
+                null=True,
+                db_constraint=False,  # Adding this manually to prevent error when deleting books
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="introduction_for_historical_book",
+                to="research.textobjectfield",
+            ),
+        ),
+        migrations.AddField(
+            model_name="historicalwork",
+            name="introduction",
+            field=models.ForeignKey(
+                null=True,
+                db_constraint=False,  # Adding this manually to prevent error when deleting works
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="introduction_for_historical_work",
+                to="research.textobjectfield",
+            ),
+        ),
+    ]
```

Comment:
So, when I started this up on my machine, django logs said there were changes to models that needed migrating. Make migrations brought up changes to Antiquarian.introduction, HistoricalBook.introduction, and HistoricalWork.introduction. I get the Antiquarian one because related name has changed, but historical changes are a mystery. In any case, adding 0064 migration should suffice.

Reply:
We should probably have that as a CI test: i.e. it fails if running makemigrations creates a new migration file. I'll create an issue.